app/src/main/java/com/example/polina/socialnetwork/ProfileFragment.java

```diff
@@ -32,31 +34,49 @@
     public TextView birthday;
     public NetworkImageView image;
     public TextView name;
+    Button postSend;
+    Button followings;
+    Button followers;
     ViewGroup header;
     Context thisContext;
+    CheckBox checkBoxFollow;
 
 
     private String profileURL;
     private String connectionFailed;
     private SharedPreferences sharedPreferences;
     private String idUser;
+    private String idUserFollow;
```

Comment:
it's not necessary to use this new variable. please reuse idUser for that purposes. and rename it to userId